### PR TITLE
Ignore img tag when using linkifier

### LINKED ISSUE #2941: Exclude `img` tags from linkifyHtml
**What would you like to be added**:
The `img` tag needs to be excluded when we call `linkifyHtml` in https://github.com/Seneca-CDOT/telescope/blob/f80b0b9f673b51cded172c91719e4c0b564d31fe/src/backend/data/post.js#L158
**Why would you like this to be added**:
Right now all the `img` tags inside a post content are wrapped by `a` tags, this results in the images acting like an `a` tag (open as URL on click) and also interferes with the new image zoom feature, more specifically this issue pointed out by @TueeNguyen https://github.com/Seneca-CDOT/telescope/pull/2932#pullrequestreview-883265391

### PR DESCRIPTION
**Pull request by tpmai22**
## Issue This PR Addresses
Fixed #2941 

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description
Ignore `img` tag when `linkify`

## Steps to test the PR
At the `root` directory remove `redis-data`

```
rm -r redis-data
```

Rebuild images, `pnpm services:start` and wait for `redis` to cache the feeds 

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by humphd — APPROVED** [review]

File: src/api/parser/src/data/post.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ class Post {
     }
 
     // Wrap an <a> tag on any link inside our content
-    article.content = linkifyHtml(article.content);
+    article.content = linkifyHtml(article.content, { ignoreTags: ['img'] });
```
Comment (DukeManh) [inline src/api/parser/src/data/post.js:154]: Good job, I think you need to remove the current temporary fix for this as well. cc @416rehman

Comment (DukeManh) [reply src/api/parser/src/data/post.js:154]: You can safely remove this if this works
https://github.com/Seneca-CDOT/telescope/blob/c29eec4be6cee6aaf6ade6186fa7c0942d61bc2d/src/web/src/components/Posts/Post.tsx#L330

Comment (tpmai22) [reply src/api/parser/src/data/post.js:154]: @DukeManh just upload another commit

**Review by DukeManh — APPROVED** [review]

**Review by tcvan0707 — APPROVED** [review]

File: src/api/parser/src/data/post.js (hunk 1 above)
Comment (TueeNguyen) [reply src/api/parser/src/data/post.js:154]: nice catch

**Review by TueeNguyen — APPROVED** [review]

**Pull request merged**